nav2_simple_commander/nav2_simple_commander/costmap_2d.py

```diff
@@ -71,3 +71,72 @@ def getGlobalFrameID(self):
     def getCostmapTimestamp(self):
         """Get costmap timestamp."""
         return self.costmap_timestamp
+
+    def getCost(self, _index: int = None, _mx: int = None, _my: int = None) -> np.int8:
```

Comment:
Create 2 separate implementations for index vs mx,my to have API parity with C++ usage. No reason to confuse users having to `null` out `index` to use the mx/my API

Reply:
I understand your comment as "split as two seperate functions like getCostFromIndex and getCostFromCell" 
Is it right?
I want to adopt the method overloading, but python does not support naturally.
If I use the multipledispatch package, I can use method overloading. 
I checked this post 
https://www.geeksforgeeks.org/python-method-overloading/amp/
If you have another idea, please let me know little more specific.
Thank you for your comment 👍

Reply:
Ah, I did not konw that. How about `getCostIdx` and `getCostXY`?

Reply:
Okay that is better and simpler.
I will change it.
Thank you for your support 😆

Reply:
I split the getCost into getCostIdx and getCostXY